lib/src/Uncertainty/Bayesian/openturns/CalibrationResult.hxx

```diff
@@ -134,6 +135,17 @@ private:
   Sample outputAtPriorMean_;
   Sample outputAtPosteriorMean_;
 
+  /* Bayesian */
+  Bool bayesian_;
+
+  /* Colors */
+  String priorLineStyle_;
+  String posteriorLineStyle_;
+  String observationLineStyle_;
+  String priorColor_;
+  String posteriorColor_;
+  String observationColor_;
```

Comment:
This can be easily done for line styles: I fixed that. Thank you for the advice. For colors, this is a different story. This is because some draw() methods do not use the `observationColor`. Hence, if the 3 colors are not attributes of the object, I cannot make sure that the same colors are used in all objects. This is why I maintain the color attributes, and add them to the save/load methods.

@jschueller Are you OK with that?

Reply:
absolutely

Reply:
Ok.